src/hooks/useBalanceChartData/useBalanceChartData.ts

```diff
@@ -341,14 +341,14 @@ type UseBalanceChartData = (args: UseBalanceChartDataArgs) => UseBalanceChartDat
   balances and fiat prices for each time interval (bucket) of the chart
 */
 export const useBalanceChartData: UseBalanceChartData = args => {
-  const { assetIds: inputAssetIds, accountId, timeframe } = args
+  const { assetId, accountId, timeframe } = args
   const assets = useAppSelector(selectAssets)
-  const accountIds = useMemo(() => (accountId ? [accountId] : []), [accountId])
   const [balanceChartDataLoading, setBalanceChartDataLoading] = useState(true)
   const [balanceChartData, setBalanceChartData] = useState<BalanceChartData>(makeBalanceChartData())
 
+  const filter = useMemo(() => ({ accountId }), [accountId])
```

Comment:
this was just a bug that i introduced a few days ago, we need to filter by account id if it's passed in